contracts/src/node-interface/NodeInterface.sol

```diff
@@ -50,17 +49,18 @@ interface NodeInterface {
 
     /**
      * @notice Finds the L1 batch containing a requested L2 block, reverting if none does
-     * @param block The L2 block being queried
+     * Throws if block doesn't exist, or if block number is 0
+     * @param blockNum The L2 block being queried
      * @return batch The L1 block containing the requested L2 block
      */
-    function findBatchContainingBlock(uint64 block) external view returns (uint64 batch);
+    function findBatchContainingBlock(uint64 blockNum) external view returns (uint64 batch);
 
     /**
      * @notice Gets the number of L1 confirmations of the sequencer batch producing the requested L2 block
      * This gets the number of L1 confirmations for the input message producing the L2 block,
      * which happens well before the L1 rollup contract confirms the L2 block.
      * @param blockHash The hash of the L2 block being queried
-     * @return confirmations The number of L1 confirmations the sequencer batch has
+     * @return confirmations The number of L1 confirmations the sequencer batch has. Returns 0 if block does not exist.
```

Comment:
It reverts with `unknown block hash` if the block doesn't exist, however if the block exists but hasn't been put in a batch yet, then it returns 0. Still good to edit this comment like that cause those aren't straightforward details :)

Reply:
Updated